### PR TITLE
fix notice

### PR DESCRIPTION
**Pull request by nguyenk**

### CODE REVIEW

File: src/Mouf/Security/UserService/UserService.php [hunk 1]
```diff
@@ -159,7 +159,7 @@ public function login($login, $password) {
 	 */
 	public function loginWithoutPassword($login) {
 		// First, if we are logged, let's unlog the user.
-		if (!$this->byPassIsLogged && $this->isLogged()) {
+		if ((!$this->byPassIsLogged || !isset($this->byPassIsLogged)) && $this->isLogged()) {
```
Comment (moufmouf) [inline src/Mouf/Security/UserService/UserService.php]: You should never perform a "isset" on a $this->xxx field...
However, the field should ALWAYS be declared in the class, and it is not...

Comment (nguyenk) [inline src/Mouf/Security/UserService/UserService.php]: Ok, so you mean I should remove the isset() an declare the parameter in the class ?

Comment (moufmouf) [inline src/Mouf/Security/UserService/UserService.php]: Yep :)

**Comment by nguyenk** [comment]
Ok, done !

**Pull request merged**